enrollment-server-onboarding/src/main/java/com/wultra/app/onboardingserver/impl/service/document/DocumentProcessingService.java

```diff
@@ -369,6 +373,7 @@ private void processDocsSubmitResults(OwnerId ownerId, DocumentVerificationEntit
         if (docSubmitResult.getErrorDetail() != null) {
             docVerification.setStatus(DocumentStatus.FAILED);
             docVerification.setErrorDetail(docSubmitResult.getErrorDetail());
+            docVerification.setErrorOrigin(ErrorOrigin.DOCUMENT_VERIFICATION);
         } else if (docSubmitResult.getRejectReason() != null) {
             docVerification.setStatus(DocumentStatus.REJECTED);
             docVerification.setRejectReason(docSubmitResult.getRejectReason());
```

Comment:
Here we should obviously set also the `errorOrigin`?

Basically if there will be only the `errorOrigin` we should use it at all places where we set the `rejectReason` too.